recipes/py-gfm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "py-gfm" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "f107e43248eba6236b19ceda78531f40c7bdb85ba4a219f478c92920397f4f98" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed
+  
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - markdown
```

Comment:
It looks like `markdown` should be in the `requirements/run` section since it's needed to import the package. The `test/requires` would be things that are required to run a test suite that aren't core dependencies.